src/main/java/network/brightspots/rcv/CommonDataFormatReader.java

```diff
@@ -132,6 +132,25 @@ void parseXml(List<CastVoteRecord> castVoteRecords)
       CastVoteRecordReport cvrReport = xmlMapper.readValue(inputStream, CastVoteRecordReport.class);
       inputStream.close();
 
+      // Some checks to provide nicer error messages.
+      // This is common with Unisyn's CDF CVR, which is not compatible with RCTab.
+      if (cvrReport.GpUnit == null) {
+        Logger.severe("Field \"GPUnit\" missing from CVR. "
```

Comment:
Nit: I think we generally put an exclamation point for `severe` log messages. E.g. here would be "... missing from CVR!". Below would be "... missing from CDF CVR file!".

Also recommend consistency between these messages, i.e. first would say "...missing from CDF CVR file!".